File: MAP/LAB7-MAP/LAB7-MAP/Expressions/ConstExpr.cs
using MAP_Lab7.Model;

namespace MAP_Lab7.Expressions
{
    public class ConstExpr : IExpression
    {
        private int cnst;

        public ConstExpr(int c)
        {
            cnst = c;
        }

        public int Eval(IDictionaryy<string, int> dict)
        {
            return cnst;
        }

        public override string ToString()
        {
            return " " + cnst;
        }
    }
}

File: MAP/Lab3/src/Model/Expression/ConstExpr.java
package Model.Expression;

import Model.IDictionary;

public class ConstExpr implements Expression
{
    private int value;
    public ConstExpr(int v)
    {
        value = v;
    }

    @Override
    public int Eval(IDictionary<String, Integer> d)
    {
        return value;
    }

    @Override
    public String toString()
    {
        return "" + value;
    }
}
